groupie/src/test/java/com/genius/groupie/SectionTest.java

```diff
@@ -565,4 +583,22 @@ public void insertGroupToNestedSectionNotifiesAtCorrectIndex() throws Exception
         rootSection.add(1, nestedSection3);
         verify(groupAdapter).onItemRangeInserted(rootSection, 2, 2);
     }
+
+    @Test
+    public void removeGroupFromNestedSectionNotifiesAtCorrectIndex() throws Exception {
+        final Section rootSection = new Section();
+
+        rootSection.setGroupDataObserver(groupAdapter);
```

Comment:
It does, but groupAdapter in this case is mocked so the add() method does nothing.

Of course, that also means that the call to add() on the next line is unnecessary. Do you want me to remove it?